src/App.tsx

```diff
@@ -1,14 +1,62 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { ChangeEvent, FC, useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/FilterBy';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getFilteredTodos } from './utils/getFilteredTodos';
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.all);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [query, setQuery] = useState('');
+
+  const handleChangeFilter = (event: ChangeEvent<HTMLSelectElement>) => {
```

Comment:
You are not following the recommended naming conventions for event handler functions. Instead of `handleChangeFilter`, consider using `onFilterChange` to make it clear that this is an event handler function.